ui/desktop/app/templates/application.hbs

```diff
@@ -21,30 +21,35 @@
 
       <header.utilities as |utilities|>
         {{#if this.showWindowActions}}
-          <Hds::Button
-            @icon='x'
-            @size='small'
-            @text={{t 'actions.close'}}
-            @color='critical'
-            class='button-window-close'
-            {{on 'click' this.close}}
-          />
-          <Hds::Button
-            @icon='x'
-            @size='small'
-            @color='secondary'
-            @text={{t 'actions.fullscreen'}}
-            class='button-window-fullscreen'
-            {{on 'click' this.toggleFullScreen}}
-          />
-          <Hds::Button
-            @icon='x'
-            @size='small'
-            @color='secondary'
-            @text={{t 'actions.minimize'}}
-            class='button-window-minimize'
-            {{on 'click' this.minimize}}
-          />
+          <Hds::ButtonSet>
+            <Hds::Button
+              @icon='monitor'
+              @text={{t 'actions.fullscreen'}}
+              @size='small'
+              @color='secondary'
+              @isIconOnly={{true}}
```

Comment:
Note: Even though `iconOnly` is `true`, we still need to define the `@text` value because it'll fn as the aria-label attribute on the btn.